server/routerlicious/README.md

```diff
@@ -119,12 +119,6 @@ If you want to build API documentation locally, see [Building Documentation](htt
 
 ## Architecture
 
-Below is the original Routerlicious architecture diagram. The current system has slight changes from the diagram but
-largely remains unchanged. Detailed descriptions of the components are contained below as well as callouts
-to areas that have changed from the original picture. We will update the README with a more current diagram soon.
-
-![Routerlicious architecture diagram](../../docs/architecture/server/SystemArchitecture.jpg)
```

Comment:
I wasn't able to find this image within the repo - let me know if it's still around.

Reply:
Yeah. That file was moved in PR #1391 and that deleted in #3087.
The diagram is quite old and doesn't reflect the current picture. We should create a new one an d I have create #5002 to track that.  Removing it for now is good.